src/stores/osimRuntime.ts

```diff
@@ -27,21 +27,29 @@ const OsimRuntime = z.object({
     osidb: z.string(),
     bugzilla: z.string(),
   }),
-  osimVersion: z.string(),
+  osimVersion: z.object({
+    rev: z.string(),
+    tag: z.string(),
+    timestamp: z.string(),
+    dirty: z.boolean(),
+  }),
+  env: z.string().default(''),
   error: z.string().default(''),
+  readOnly: z.boolean().default(false),
```

Comment:
[docs](https://github.com/RedHatProductSecurity/osim/blob/feature/OSIDB-2317-read-only-mode/docs/overview.md#local-configuration) local config example should be refined to comply with the changes.